modules/Microsoft.Sql/servers/deploy.bicep

```diff
@@ -77,6 +77,14 @@ param privateEndpoints array = []
 ])
 param publicNetworkAccess string = ''
 
+@description('Optional. Whether or not to restrict outbound network access for this server.')
+@allowed([
+  ''
+  'Enabled'
+  'Disabled'
+])
+param restrictOutboundNetworkAccess string = ''
```

Comment:
Can you please update one of the test files to validate usage of this parameter: [Example - Common](https://github.com/Azure/ResourceModules/blob/main/modules/Microsoft.Sql/servers/.test/common/deploy.test.bicep). This helps knowing if it conflicts with other parameters for this module.

Reply:
Added the default value from a portal deployment to the test.